components/prism-diff.js

```diff
@@ -39,11 +39,28 @@
 			alias.push("bold");
 		}
 
-		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
-			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
-		};
+		var pattern = RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm');
+
+		if (['-', '<', '+', '>'].indexOf(prefix) !== -1) {
+			var inside = {
+				line: {
+					pattern: /^(.).+(?:\r\n?|\n)?/,
+					lookbehind: true
+				},
+				prefix: /[\s\S]/
```

Comment:
Please add single quotes around the `prefix` and `line` properties.

We do this to differentiate Prism grammar tokens from normal object properties. This is just a small convention to make language definitions easier to read.

```suggestion
				'prefix': /[\s\S]/
```